.specify/scripts/bash/common.sh

```
@@ -0,0 +1,125 @@
+get_feature_paths() {
+    local repo_root=$(get_repo_root)
+    local current_branch=$(get_current_branch)
+    local has_git_repo="false"
+
+    if has_git; then
+        has_git_repo="true"
+    fi
+
+    local specs_base="${SPECIFY_SPECS_DIR:-.specify/specs}"
+    local feature_dir_name="${SPECIFY_FEATURE_DIR:-$current_branch}"
+    local feature_dir=$(get_feature_dir "$repo_root" "$feature_dir_name")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)